features/bounding-box-zoeken.feature

```diff
@@ -0,0 +1,131 @@
+# language: nl
+
+Functionaliteit: Als gebruiker wil ik een pand of adresseerbaar object kunnen zoeken binnen een gebied
+    Zodat ik de adresseerbare objecten of panden kan vinden binnen een rechthoek die ik teken op een kaart
+
+    Zoeken in een rechthoekig gebied kan met de parameter bbox.
+
+
+Rule: De standaardwaarde voor het coördinatenstelsel (CRS) voor de bbox-parameter is Rijksdriehoekscoördinaten New Nederlands
+
+    Scenario: zoek op bbox zonder het coördinatenstelsel aan te geven
+        Gegeven het pand met identificatie "0826100000036343" heeft de volgende geometrie polygoon coördinaten
+        | geometriecoördinaten                                                                                                        |
+        | [ [118315.606,404844.967],[118324.183,404843.509],[118325.179,404849.365],[118316.604,404850.835],[118315.606,404844.967] ] |
+        Als met "GET" "/panden?bbox=118314,404843,118326,404850" wordt gezocht zonder request header "Content-Crs"
```

Comment:
request header is hier met spatie. Bij locatie-zoeken is het requestheader zonder spatie. Als er voor één manier wordt gekozen is het niet nodig om automation code te dupliceren